src/test/java/uk/gov/hmcts/sptribs/controllers/CaseManagementControllerTest.java

```diff
@@ -45,107 +48,111 @@ class CaseManagementControllerTest {
     private CaseManagementController caseManagementController;
 
     @Mock
-    CaseManagementService caseManagementService;
+    private CaseManagementService caseManagementService;
+
+    private AutoCloseable closeableMocks;
 
     @BeforeEach
     void setUp() {
-        MockitoAnnotations.openMocks(this);
+        closeableMocks = MockitoAnnotations.openMocks(this);
+    }
+
+    @AfterEach
+    void tearDown() throws Exception {
+        closeableMocks.close();
     }
 
     @Test
     void testCicCreateCaseData() throws Exception {
```

Comment:
Add a test cases for the blow,
1. create case data fails with exception
2. update case data fails with exception
3. fetch case details fails with exception